### PR TITLE
TDP-2811 update voice url

### PR DESCRIPTION
**Pull request by rmello-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [x] I made sure that the site looks great and functions perfectly live
- [x] I ran splell check
- [x] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [x] I did these things
- [x] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-2811-update-voice-url.s3-website-us-east-1.amazonaws.com/

**Comment by muesono-daitan** [comment]
There are urls referencing api.catapult.inetwork.com. I don't know if there is a jira for this or if it should be fixed in this PR though.

**Comment by jmulford-bandwidth** [comment]
> There are urls referencing api.catapult.inetwork.com. I don't know if there is a jira for this or if it should be fixed in this PR though.

We caught this in DX-537 (stopgap docsite audit) and will be making a separate jira for it. Thanks for reassuring it!

**Comment by rmello-daitan** [comment]
> > There are urls referencing api.catapult.inetwork.com. I don't know if there is a jira for this or if it should be fixed in this PR though.
> 
> We caught this in DX-537 (stopgap docsite audit) and will be making a separate jira for it. Thanks for reassuring it!

Oh, I was going to fix it. It is just in the README.md file. We need to fix the "Make a call" examples as well. They are missing `answerUrl` and `applicationId`

**Comment by jmulford-bandwidth** [comment]
> > > There are urls referencing api.catapult.inetwork.com. I don't know if there is a jira for this or if it should be fixed in this PR though.
> > 
> > 
> > We caught this in DX-537 (stopgap docsite audit) and will be making a separate jira for it. Thanks for reassuring it!
> 
> Oh, I was going to fix it. It is just in the README.md file. We need to fix the "Make a call" examples as well. They are missing `answerUrl` and `applicationId`

I just spoke with @dtolb and we're gonna revamp that whole home page for this launch so no need to update that

**Comment by dtolb** [comment]
> > > > There are urls referencing api.catapult.inetwork.com. I don't know if there is a jira for this or if it should be fixed in this PR though.
> > > 
> > > 
> > > We caught this in DX-537 (stopgap docsite audit) and will be making a separate jira for it. Thanks for reassuring it!
> > 
> > 
> > Oh, I was going to fix it. It is just in the README.md file. We need to fix the "Make a call" examples as well. They are missing `answerUrl` and `applicationId`
> 
> I just spoke with @dtolb and we're gonna revamp that whole home page for this launch so no need to update that

https://github.com/Bandwidth/bandwidth.github.io/pull/179 Covers changes

File: voice/bxml/callBacks/answer.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ POST http://[External server URL]
 	"to":"+15553334444",
 	"direction":"outbound",
 	"callId":"3C5F8A8B-CDA2-443A-B797-B3ADA2197E05",
-	"callUrl": "https://voice.bandwidth.com/api/v2/users/55555555/calls/3C5F8A8B-CDA2-443A-B797-B3ADA2197E05"
```
Comment (dtolb) [inline voice/bxml/callBacks/answer.md]: See the new TDP-2824

**Review by dtolb — CHANGES_REQUESTED** [review]
@rmello-daitan 

Per TDP-2824 the voice url in it's entirety should be 
`https://voice.bandwidth.com/api/v2/accounts/{{accountId}}/calls`

and then `https://voice.bandwidth.com/api/v2/accounts/{{accountId}}/calls/{{callId}}`

There are a few places in this PR where the old url had `.../api/v2/users/` that were fixed that should be updated correctly.

**Comment by rmello-daitan** [comment]
> @rmello-daitan
> 
> Per TDP-2824 the voice url in it's entirety should be
> `https://voice.bandwidth.com/api/v2/accounts/{{accountId}}/calls`
> 
> and then `https://voice.bandwidth.com/api/v2/accounts/{{accountId}}/calls/{{callId}}`
> 
> There are a few places in this PR where the old url had `.../api/v2/users/` that were fixed that should be updated correctly.

yep, just saw it. I'll update the code

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-2811-update-voice-url.s3-website-us-east-1.amazonaws.com/

**Review by jmulford-bw — APPROVED** [review]

**Review by dtolb — APPROVED** [review]

**Review by mrmeyers99 — APPROVED** [review]
